configure.ac

```diff
@@ -929,15 +954,29 @@ AS_IF([test x$with_pam != xno],
         [have_pam=no]
   )
   AC_CHECK_FUNCS(fgetpwent fgetgrent)
-  AS_IF([test "x$have_pam" = "xyes" && \
-      test "x$have_userprogs" = "xyes" && \
-      test "x$ac_cv_func_fgetpwent" = "xyes" && \
-      test "x$ac_cv_func_fgetgrent" = "xyes"],
-  [
-      users_promises_ok=yes
-  ],[
-      users_promises_ok=no
-  ])
+  AS_IF([test "x$ac_cv_func_fgetpwent" = "xyes" && \
+         test "x$ac_cv_func_fgetgrent" = "xyes"],
+         [have_fgetpwent_fgetgrent=yes]
+         [AC_DEFINE(HAVE_FGETPWENT_FGETGRENT, 1, [Define if fgetpwent and fgetgrent functions are present])],
+         [have_fgetpwent_fgetgrent=no]
+     )
+
+     AS_CASE(["$target_os"],
```

Comment:
Please fix the indentation here to make it easier to see which macros are at the same level.

Reply:
fixed